src/Symfony/Component/HttpClient/HttpClientTrait.php

```diff
@@ -498,6 +499,10 @@ private static function resolveUrl(array $url, ?array $base, array $queryDefault
             $url['query'] = null;
         }
 
+        if (null === $url['authority']) {
+            throw new InvalidArgumentException('Invalid URL: host is missing.');
```

Comment:
let's make the message closer in style to the existing ones?

Reply:
What do you suggest instead?

Reply:
```suggestion
            throw new InvalidArgumentException(sprintf('Missing host in "%s".', $givenUrl));
```

Reply:
I see. Well, the current message would be fine for me then.

Reply:
I think I found a way to improve the message.